frontend/src/pages/RunDetails.test.tsx

```diff
@@ -1139,10 +1158,11 @@ describe('RunDetails', () => {
             className="page"
           >
             <Banner
-              additionalInfo="getting logs failed"
-              message="Warning: failed to retrieve pod logs. Possible reasons include cluster autoscaling or pod preemption"
-              mode="warning"
+              additionalInfo="Possible reasons include cluster autoscaling, pod garbage collection and pod preemption. Error response: pod not found"
```

Comment:
nit: let's order them based on likelihood?
pod garbage collection > cluster autoscaling ~= pod preemption, what do you think?